druid/druid.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script installs Druid on a Google Cloud
+# Dataproc cluster.
+#
+# To use this script, you will need to configure the following variables to
+# match your cluster. For information about which software components
+# (and their version) are included in Cloud Dataproc clusters, see the
+# Cloud Dataproc Image Version information:
+# https://cloud.google.com/dataproc/concepts/dataproc-versions
+
+set -euxo pipefail
+
+export DRUID_VERSION='0.12.3'
+export DRUID_DIR="/root/druid-${DRUID_VERSION}"
+
+function err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $@" >&2
+  return 1
+}
+
+function download_druid() {
+  cd /root
+  curl -O http://static.druid.io/artifacts/releases/druid-${DRUID_VERSION}-bin.tar.gz
+  tar -xzf druid-${DRUID_VERSION}-bin.tar.gz
+}
+
+function download_samples() {
+  cd ${DRUID_DIR}
+  curl -O http://druid.io/docs/${DRUID_VERSION}/tutorials/tutorial-examples.tar.gz
+  tar zxvf tutorial-examples.tar.gz
+}
+
+function prepare_environment_services(){
+  systemctl start mysql
```

Comment:
Is it possible to support Cloud SQL? https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/tree/master/cloud-sql-proxy